Add Hono guide (#12710)
getsentry/sentry-docs: docs/platforms/javascript/common/configuration/tree-shaking.mdx

`__RRWEB_EXCLUDE_SHADOW_DOM__`

> Replacing this flag with `true` will tree shake any SDK code related to capturing shadow dom elements with Session Replay. It's only relevant when using <PlatformLink to="/session-replay">Session Replay</PlatformLink>. Enable this flag if you don't want to record any shadow dom elements. This has no effect if you did not add `replayIntegration`.

Reviewer: ```suggestion
Replacing this flag with `true` will tree shake any SDK code related to capturing shadow dom elements with Session Replay. It's only relevant when using <PlatformLink to="/session-replay">Session Replay</PlatformLink>. Enable this flag if you don't want to record any shadow dom elements. This has no effect if you didn't add `replayIntegration`.
```
